library/spdm_responder_lib/libspdm_rsp_key_pair_info.c

```diff
@@ -0,0 +1,157 @@
+/**
```

Comment:
File should be `libspdm_rsp_get_key_pair_info.c`.

Reply:
Thanks. I have changed the file name.